### PR TITLE
Orange Albatross Lesson-1-1 Assignment

### PR DESCRIPTION
**Pull request by myang1010**
This PR adds a todo list to the main `App` component

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(
```
Comment (raquii) [inline src/index.js:5]: I am almost positive this file is auto generated this way when you run `npx create-react-app` with the `--template minimal` flag, but as of React 18, `ReactDOM.render()` is no longer supported.

You can change this file to instead use the new `createRoot` function, like so: 

```js
import { createRoot } from 'react-dom/client';

const container = document.getElementById('root');
const root = createRoot(container);

root.render(<App />);
```

📖 If you'd like to read more about this change, you can check out this blog post from React: [Updates to Client Rendering APIs](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis)

**Review by raquii — COMMENTED** [review]
Great work! A tiny comment about a recent change to React that unfortunately hasn't been reflected in the CRA template you were told to use, just for your own edification. 

The other piece that I would say is about your PR descriptions-- In general, we want our descriptions to be as thorough as possible. While "Complete lesson-1-1 coding assignment" is definitely accurate, we could improve our description by being more specific about _what lesson-1-1 actually is._

In a job setting, PRs often come as a result of a "ticket" in a project management tool and developers are working on their own tickets while being asked to review code for tickets that they aren't familiar with the description of. If I put up a PR that said "Completed SWA-123", my coworkers would have to go look up my ticket before they could review my code, which they would probably find irritating. 😅 

Instead, describe what the objective of the changes were. It could be something as simple as "This PR adds a todo list to the main `App` component" or more descriptive if you wish! The more changes, the more detail you should add.

File: src/index.js (hunk 1 above)
Comment (myang1010) [reply src/index.js:5]: Thank you Raquel for the review and suggestion. Will update the file in my next pull request.

**Comment by myang1010** [comment]
> Instead, describe what the objective of the changes were. It could be something as simple as "This PR adds a todo list to the main `App` component" or more descriptive if you wish! The more changes, the more detail you should add.

Thank you! I will make my PR description more specific next time. 😃

**Pull request merged**